File: node/models/company.js
/**
 * Created by nidhi on 16/12/13.
 */
function company(){
    this.name = null
    this.shortname = null;
    this.rate = null;
    this.previous = null;
    this.change = null;
    this.supply = null;
}

exports.create =  function () {
    var instance = new company();
    return instance;
}

File: android/ECellVSM/app/src/main/java/com/aasim/ecellvsm/Models/Company.java
package com.aasim.ecellvsm.Models;

import java.util.ArrayList;

/**
 * Created by aasim on 26/12/16.
 */

public class Company {
    String _id,name,shortname;
    float rate,previous,change;

    public String get_id() {
        return _id;
    }

    public String getName() {
        return name;
    }

    public String getShortname() {
        return shortname;
    }

    public float getRate() {
        return rate;
    }

    public float getPrevious() {
        return previous;
    }

    public float getChange() {
        return change;
    }
}
